File: Codeforces/1742A.py
n = int(input())

a=0
b=0
c=0

for _ in n:
    a = int(input())
    b = int(input())
    c = int(input())
    
    if a+b == c:
        print("YES")
    elif a + c == b:
        print("YES")
    elif b+c == a:
        print("YES")
    else:
        print("NO")

File: Codeforces/1742A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int t;
    cin >> t;

    while (t--) {
        int a, b, c;
        cin >> a >> b >> c;

        if (a + b == c || a + c == b || b + c == a) {
            cout << "YES\n";
        } else {
            cout << "NO\n";
        }
    }

    return 0;
}
